pkg/volume/csi/csi_plugin.go

```diff
@@ -213,15 +208,9 @@ func (p *csiPlugin) Init(host volume.VolumeHost) error {
 	p.host = host
 
 	if utilfeature.DefaultFeatureGate.Enabled(features.CSIDriverRegistry) {
-		csiClient := host.GetKubeClient()
-		if csiClient == nil {
-			return errors.New("unable to get Kubernetes client")
+		if p.host.GetCSIDriverLister() == nil {
```

Comment:
I don't think we need this check anymore. We can check for an unset lister right before we need to use it.

Reply:
Ok